index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
- [ ] [OPTIONAL] Kindly, make sure your website looks exactly like the design (`font size, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.).

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture1](https://user-images.githubusercontent.com/57408419/234747618-3ffe2235-9385-4ca9-a20d-6b10a901be25.PNG)| |![Capture](https://user-images.githubusercontent.com/57408419/234747326-7db71159-6233-4455-9ae9-1affa8a9a7a8.PNG)| |